docs/workflow/blocks/http.rst

```diff
@@ -52,7 +52,11 @@ Examples
 
 
 **Dynamic data** If the endpoint needs to be constructed from data, the endpoint can be specified as an object with a "valueGetter" attribute.
-"valueGetter" can only get data from the context.
+"valueGetter" can only get data from the context, it does not work with a static data. If values come from context, they must be specify inside the "valueGetter"
+property. If contect is used in other property of the "endpoint", it will not grab any context value.
```

Comment:
Let's not say "grab". If a word other than "get" is needed, "retrieve" is a good one.

Reply:
@lukestanley Why you suggest not to use "grab"? What is the issue with that term? (genuine question)

Reply:
Well `grab` is quite informal, though it is a typical word to use in discussion, for a consistent style between the documents, there are more typical words instead of `grab` that would usually be used in the context of a technical document. By the way, ChatGPT says:
`The term "grab" can be seen as informal or colloquial, which might not be appropriate for technical documentation, especially if the document is intended for a diverse or global audience. It can also be vague, as it doesn’t specify the exact nature of the action being described. For instance, does "grab" mean "fetch," "retrieve," "download," or something else? Using more precise terms helps to maintain clarity and professionalism in the documentation.` https://chatgpt.com/share/e1d67316-3aca-4d4a-b20e-6fcc75c71fa5
@gsambrotta

Reply:
Ohhh very interesting @lukestanley Thank you for the explanation